ste/sender-blockBlobFromURL.go

```diff
@@ -68,20 +68,15 @@ func (c *urlToBlockBlobCopier) GenerateCopyFunc(id common.ChunkID, blockIndex in
 		setPutListNeed(&c.atomicPutListIndicator, putListNotNeeded)
 		return c.generateCreateEmptyBlob(id)
 	}
-
-	if c.NumChunks() == 1 && adjustedChunkSize <= int64(azblob.BlockBlobMaxUploadBlobBytes) &&
-		c.blockBlobSenderBase.jptm.FromTo() == common.EFromTo.GCPBlob() {
+	// Small blobs from all sources will be copied over to destination using PutBlobFromUrl with the exception of files
+	fromTo := c.blockBlobSenderBase.jptm.FromTo()
+	if c.NumChunks() == 1 && adjustedChunkSize <= int64(azblob.BlockBlobMaxUploadBlobBytes) && fromTo.From() != common.ELocation.File() {
```

Comment:
Just curious: Are we only disabling `fromTo.From() != ELocation.File()` ? I thought it was `fromTo.To() != ELocation.File()` as well. Please confirm once.

Reply:
This code executes only when destination is blobstore.

Reply:
Ohh you're right.